source/monitoring_audience/analyzing_data_usage.rst

```diff
@@ -1,90 +1,109 @@
 Analyzing data usage
 ====================
 
-Go to the *Analytics > Data* page.
+The Data dashboard displays statistics to analyze the data published on the portal. It is available in the Analytics > Data section of the back office.
 
-There are seven indicators :
+This dashboard is separated into 4 sections:
 
-1. **Dataset activity**: Download and API calls of most used datasets. Please note that there are two different scales for the downloads count and the number of API calls on each side of the chart.
+- Totals: figures about the datasets published in the portal and their usage through that portal
+- Popularity: statistics and charts to compute and analyze the popularity of the datasets published in the portal
+- Top 5: lists of most downloaded, used and reused published datasets
+- Theme Distribution: tree chart to analyze most and least used themes
 
-.. ifconfig:: language == 'en'
+.. admonition:: Datasets used to build the Data dashboard
+   :class: important
 
-    .. image:: usage__data-usage-1-en.jpg
-        :alt: Data Usages Popularity Indicator
+   In this dashboard, only published datasets are taken into account. Unpublished datasets are ignored from all figures, charts and other computed analytics displayed.
 
-.. ifconfig:: language == 'fr'
+   It is however possible, by clicking the buttons at the top of the dashboard, to choose to build the dashboard:
 
-    .. image:: usage__data-usage-1-fr.jpg
-        :alt: Usage Data Indicateur Popularité
+   - with all existing published datasets (checked by default)
+   - only with the private datasets, meaning those in restricted access, no matter the restriction
+   - or, only with the public datasets, meaning those accessible by everyone.
 
-2. **Total datasets**: Total number of datasets in the catalog.
+   Depending on which datasets are used to build the Data dashboard, the displayed figures, charts and analytics will vary.
```

Comment:
This could be tied to the choice made above? Something such as "Depending on the selected option, the displayed figures..." or "The displayed figures, charts and analytics will vary according to the selected option"

Reply:
Actually no, it makes sense like this.